python/tvm/relay/backend/vm.py

```diff
@@ -178,6 +178,11 @@ def compile(self, mod, target=None, target_host=None):
         """
         target = _update_target(target)
         target_host = None if target_host == "" else target_host
+        if not target_host:
```

Comment:
duplicate if condition with line 182

Reply:
After this if block, it's still possible that `target_host` is None. So it's necessary to keep the if condition in line 186